### PR TITLE
fixing github page

### PR DESCRIPTION
**Pull request by aron-net**
I try to make GitHub page and I marge it by mistake

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        </div>
+        <div class="partner-home">
+            <h2 class="partner-header">Partner</h2>
+            <hr class="line-1">
+            <div class="partner-image">
+                <img class="mozilla" src="./Image/Partner/mozilla.png" alt="mozilla-logo">
+                <img class="google" src="./Image/Partner/google-logo.png" alt="google-logo">
+                <img class="microverse" src="./Image/Partner/5e30ae0c4081806a8f50a7df_horizontal-logo-white.png" alt="Microverse-logo">
+                <img class="spotify" src="./Image/Partner/Spotify_Logo_CMYK_Green.png" alt="spotify-log">
+                <img class="lumen" src="./Image/Partner/lumen.png" alt="lumen-logo">
```
Comment (Davidosky007) [inline index.html]: * @aron-helu kindly fix all the broken images and icons on your website as i can see your project is missing the image folder you are trying to import from. below ⬇️ is a screenshot of how your webpage looks now 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @aron-helu 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
* Your deployment to Github pages still doesnt work, kindly fix this as its broken 
* your webpage is broken probably due to the fact that some file are missing from your previous Pull request kindly add them all here so that your webpage looks exactly like before 🙏  
image

_Check the comments under the review._

## Optional suggestions
* Please avoid merging/closing PRs in the future, you only do that when the PR has been approved and even if you do merge by mistake try to reference them in the new PR , To reference a PR you can post the link of the PR in the review comments or use the PR number, for example, [#2](https://github.com/microverse/guides/tse/how_to_be_a_tse/-/issues/2) (in this case [#2](https://github.com/microverse/guides/tse/how_to_be_a_tse/-/issues/2) reference PR number 2)
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aron-net** [comment]
Add the remaining files.

**Comment by aron-net** [comment]
I try to fix the error but I do not know. why it is not working. but my page has been deployed there you can check the life dom

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @aron-helu,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### ⚠️Please consider the Optional suggestions below⚠️



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- In your readme file in the Acknowledgments section please give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**